lib/generators/avo/resource_generator.rb

```diff
@@ -130,6 +130,7 @@ def generate_fields
         fields_from_model_tags
 
         generated_fields_template
+      rescue StandardError => e
```

Comment:
I think we need to rescue from a "Can't connect to DB" error.
This can silently fail from many other issues.

Also, we should rescue that only from `fields_from_model_db_columns` or methods that call the DB.

Reply:
makes sense; will do it now 👍